api/src/services/export/feedback-mapper.js

```diff
@@ -0,0 +1,49 @@
+const moment = require('moment');
+const db = require('../../db');
+const MAX_CHARS = 32767; // libre office limit of chars per cell but it seems reasonable
```

Comment:
I feel like this should be an exposed option against export, like`spreadsheet_compatibility=true` or something (the flag could be the other way as well, where you have to pass a flag to get the full results).

There are lots of ways people could consume this data, and IMO it shouldn't be bound to opening it in Office products.

We could default to "spreadsheet" support when downloading from the web UI, but leave the api supporting both?

Reply:
I'm on the fence with this one, mostly because it's an edge case. My thinking at the time was that nobody would want more than 32K anyway, so it's a sensible limitation. But your argument is convincing so I've removed the truncation for now. If it comes up in the wild we can revisit it and maybe make it standard for all fields.